EightMinuteEmpireLegends/MapLoader/CustomExceptions.cpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include "CustomExceptions.h"
+
+InvalidFileFormatException::InvalidFileFormatException() {}
+
+string InvalidFileFormatException::what() {
+	return what_message;
```

Comment:
Please be consistent in formatting, follow camel case in this line.

Reply:
I thought i removed this one already, it shouldnt be here. I will remove this.